mmcv/runner/hooks/logger/sagemaker_experiments.py

```diff
@@ -0,0 +1,84 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from ...dist_utils import master_only
+from ..hook import HOOKS
+from .base import LoggerHook
+
+
+@HOOKS.register_module()
+class SMExperimentsLoggerHook(LoggerHook):
+    """Class to log metrics to Sagemaker Experiments.
+    Works within sagemaker training job only due to library implementation.
+    It requires `sagemaker-experiments`_ and `boto3`_ to be installed.
+    Args:
```

Comment:
```suggestion

    Args:
```

Add a blank line to avoid docs format error.